source/_components/sinch.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: "Sinch SMS"
+description: "Instructions on how to add Sinch notifications to Home Assistant."
+logo: sinch.png
+ha_category:
+  - Notifications
+ha_release: 0.99.1
+redirect_from:
+ - /components/notify.sinch/
+---
+
+The `sinch` platform uses [Sinch](https://www.sinch.com/products/messaging/sms/) to deliver notifications from Home Assistant.
+
+## Prerequisites
+
+Go to your [Sinch Dashboard](https://dashboard.sinch.com/sms/api/rest) and click "Add new REST API". You should now be able to obtain your `service_plan_id` and `api_key`.
+
+## Configuration
+
+To add Sinch to your installation, add the following to your Home Assistant `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+
+# Using default recipients
+notify:
+  - platform: sinch
+    name: Sinch
+    service_plan_id: SINCH_SERVICE_PLAN_ID
+    api_key: SINCH_API_KEY
+    default_recipient: [PHONE_NO1, PHONE_NO2]
+```
+
+```
```

Comment:
This is not a code block...

Reply:
Dang, added just for readability in my editor. Fixed!